woocommerce/woocommerce
# Run API tests on PR with e2e tests

### Changes proposed in this Pull Request:

This PR adds the API Core Tests run to the workflow that runs on each PR.

Additionally, this PR adds new cleanup functions for product attributes, product categories, and product tags. It also cleans up shipping classes added in the shipping classes test. 

### How to test the changes in this Pull Request:

1. Verify CI passes (this will include the new workflow changes added on this PR)
2. Verify the syntax looks correct
3. Follow the setup for the API Core Tests in the [README file](https://github.com/woocommerce/woocommerce/tree/trunk/packages/js/api-core-tests#woocommerce-core-api-test-suite)
3. Change directories into the `plugins/woocommerce` directory
3. Run the e2e test suite and verify it passes (`pnpx wc-e2e test:e2e`)
4. After the e2e tests finish, run the API tests after and verify they pass (`pnpx wc-api-tests test api`)
